docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastForest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.Regression
+{
+    public static class FastForest
+    {
+        // This example requires installation of additional NuGet package
+        // <a href="https://www.nuget.org/packages/Microsoft.ML.FastTree/">Microsoft.ML.FastTree</a>.
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of training examples.
+            var examples = GenerateRandomDataPoints(1000);
+
+            // Convert the examples list to an IDataView object, which is consumable by ML.NET API.
+            var trainingData = mlContext.Data.LoadFromEnumerable(examples);
+
+            // Define the trainer.
+            var pipeline = mlContext.Regression.Trainers.FastForest();
+
+            // Train the model.
+            var model = pipeline.Fit(trainingData);
+
+            // Create testing examples. Use different random seed to make it different from training data.
+            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed:123));
```

Comment:
>seed:123 [](start = 91, length = 8)

why are you setting the seed here, but not above?

Reply:
I've explained it in the comment line above. We want training and test set to be different. The first one uses default seed of 0. Here I use a different seed.

---
In reply to: [266723117](https://github.com/dotnet/machinelearning/pull/2999#discussion_r266723117) [](ancestors = 266723117)

Reply:
nit: Can we be explicit to show the `seed:0` in the training case as well. Also, can `seed:123` be changed to `seed:1`? 

---
In reply to: [267018416](https://github.com/dotnet/machinelearning/pull/2999#discussion_r267018416) [](ancestors = 267018416,266723117)

Reply:
i think we're splitting hairs here. 123 is just a random number like saying ABC. 

---
In reply to: [267029054](https://github.com/dotnet/machinelearning/pull/2999#discussion_r267029054) [](ancestors = 267029054,267018416,266723117)